### ISSUE TITLE
Incorrect link on the community page

### ISSUE THREAD

**Issue by mscherer** [oVirt/ovirt-site#4]
On https://beta-ovirtweb.rhcloud.com/community/about/community-guidelines/ 

there is some example link that are not transformed to html a tag by middleman. See "To search mailing lists"?

**Comment by garrett** [comment]
I guess it's being thrown because of the `:` in the URL? Technically, it is supposed to be encoded as `%3A`... but people shouldn't have to worry about that.

So this Markdown:

```markdown
[example](https://www.google.fr/search?q=site:lists.ovirt.org+Engine+plugin)
```

Would be come this, with URL encoded string:

```markdown
[example](https://www.google.fr/search?q=site%3Alists.ovirt.org%20Engine%20plugin)
```

But, IMHO, Markdown and browsers _should_ handle things like that transparently.

**Comment by garrett** [comment]
Also, isn't it a bit odd that we're asking people to search the various resources using Google site searches like that (as Bugzilla does have a search engine)? And, there's no wiki, so that part should be removed... the site has its own search too now.

It's also bad that we're asking people to search on a different site for multiple different oVirt resources. If we were to have people search offsite for all the various resources at once, it should be prefixed with something like this, perhaps:

```
site:lists.ovirt.org OR site:ovirt.org OR (site:bugzilla.redhat.com ovirt)
```

Of course, that's a little redundant with ovirt.org being in there, with the site's own searching.

**Comment by garrett** [comment]
The above PR merge doesn't fix the issue with URLs with colons, but it does make that page better and replaces the links. (Basically, this was solved by rethinking the block instead of fixing the technical bug.)

**Comment by thatdocslady** [comment]
@garrett so what are we doing to simplify the search function? If I have anything to say about it, I vote to keep the number of search platforms as low as possible, but I don't know how you can use the ovirt site to search in bugzilla =/

**Comment by garrett** [comment]
The search function for the website only searches the website. There's no possible way to have it search anything else — it has to build an index, and it isn't going to download mailing list archives and bugzilla.

--

As far as the search form on that page, it searches both the mailing lists and bugzilla by default, but one can choose something else. I could extend that to search the website via Google too, but then it would get really noisy hits and nobody would see the issues outline in the mailing lists or bugzilla. (I've tried this.)

You can see it in action @ https://beta-ovirtweb.rhcloud.com/community/about/community-guidelines/

The embedded search form on that page, defaults search both mailing lists and bugzilla, but you can choose one or the other.

--

One possible point of "integration" across the site would be to add the functionality to send a search to Google specifically searching bugzilla and/or the mailing list by adding links on the search results page. 

(But if that's decided upon, it should be in a different, new issue, as it's unrelated to the page in question on this bug report.)

**Comment by thatdocslady** [comment]
@garrett for the sake of clarity, can you please close this issue and open a different issue that focuses on the general search functionality?

**Comment by sandrobonazzola** [comment]
@duck-rh is this still an issue? Or can we close this?

**Comment by duck-rh** [comment]
The underlying problem with URL encoding might not be solved, nevertheless this page has changed and do not contain the search form anymore.  Also the markdown_to_html() function was fixed (see #1025) and affected links on pages with a _toc_ layout like this one.

I think it's safe to close and reopen if necessary with updated information.
